.travis.yml

```diff
@@ -2,12 +2,16 @@ language: c
 sudo: required
 services: docker
 script: bash ./.travis-$BUILD_METHOD.sh
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
+    - sudo apt-get install -y python-mock python-nose
```

Comment:
We could keep this too in the script files themselves (for example I moved the "`wget https://raw.githubusercontent.com/xenserver/xenserver-build-env/master/utils/travis-build-repo.sh`" into `.travis-opam-coverage.sh`), to ensure that these packages are only installed when they are needed, only for the `python-nosetests` build method.